tmk_core/rules.mk

```diff
@@ -404,7 +404,7 @@ check-size:
 	fi
 else
 check-size:
-	echo "(Firmware size check does not yet support $(MCU) microprocessors; skipping.)"
+	$(SILENT) || echo "(Firmware size check does not yet support $(MCU) microprocessors; skipping.)"
```

Comment:
Consider using `@$(SILENT)`. Not sure if we want this or not.

Reply:
Well, this is what we use elsewhere. 
Also, this basically just quiets the message when Travis CI checks all boards, making the logs a little less verbose. 

It still shows up normally when compiling the board

Reply:
Alright, I was just going off what the remainder of the file is using, since I'm not too sure of the implications of `@` vs no `@` for our pipeline. The majority of the file seems to be using `@`, except for this section.

Reply:
Well, either way, tested, and seems to work as expected/desired.

(though, ideally, figuring out the size would be best)